nodepool/config/templates/stack-template.json

```diff
@@ -8,8 +8,50 @@
         "TargetCapacity": {{$.Worker.SpotFleet.TargetCapacity}},
         "SpotPrice": "{{$.Worker.SpotFleet.SpotPrice}}",
         "LaunchSpecifications": [
+          {{if .Worker.TopologyPrivate}}
+          {{range $subnetIndex, $subnet := $.Worker.PrivateSubnets}}
```

Comment:
@neoandroid So we now have two conditions:

1. `TopologyPrivate == true`: Worker.PrivateSubnets are used for deployment targets
2. `TopologyPrivate != true`: Worker.Subnets are used

As this introduces some duplicated code into stack-template.json, would you mind me to refactor this to:

0. Rename `Worker.Subnets` to `Worker.PublicSubnets` and remove the `if-else` blocks here.
1. `TopologyPrivate == true`: `Worker.Subnets` are populated with `Worker.PrivateSubnets`
2. `TopologyPrivate != true`: `Worker.Subnets` are populated with `Worker.PrivateSubnets`

?